doc/userguide/rules/lua-detection.rst

```diff
@@ -103,8 +111,13 @@ Entire script:
 
   return 0
 
-Sandbox and Available functions
--------------------------------
+Lua Transform: ``luaxform``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+More details in :ref:`lua-transform`.
+
+Lua Sandbox and Available functions
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
The section organization looks off right now. I think this and some others that I'll point to need adjusting:

```suggestion
Lua Sandbox and Available functions
-----------------------------------
```